tests/backup/backup_helper.go

```diff
@@ -2555,7 +2555,7 @@ func upgradeStorkVersion(storkImageToUpgrade string) error {
 		}
 	}
 	// Sleep for upgrade request to go through before validating.
-	time.Sleep(10 * time.Second)
+	time.Sleep(5 * time.Minute)
```

Comment:
can we a check for upgrade  rather than sleep?

Reply:
Done